src/Stryker.Core/Stryker.Core/Reporters/Progress/ProgressBarReporter.cs

```diff
@@ -17,12 +18,13 @@ public class ProgressBarReporter : IProgressBarReporter
         private const char ProgressBarDoneToken = '\u2588';
         private const char ProgressBarLeftToken = '-';
         private const int ProgressBarInitialState = -1;
-        private const string LoggingFormat = "Tests progress | {0} | {1} / {2} | {3} % |";
+        private const string LoggingFormat = "Tests progress | {0} | {1} / {2} | {3} % | {4} |";
 
         private readonly IConsoleOneLineLogger _testsProgressLogger;
 
         private int _totalNumberOfTests;
         private int _numberOfTestsRun;
+        private Stopwatch _startTime;
```

Comment:
I don't think the stopwatch fixes the estimate after a long sleep. It can be stopped however when the PC goes to sleep and resumed again on wake.  This could be achieved by using SystemEvents (windows only). In my opinion we don't want to use that for such a specific case. I guess it's ok for now that the estimate is a bit off after a sleep period. At least your fix makes sure the output (report) is correct.

Reply:
Well, empiric tests showed it works on my laptop. I suggest we see what happens in the large as we don't have alternative implementations.